uber/utils.py

```diff
@@ -539,6 +539,23 @@ def genpasswd():
 # Miscellaneous helpers
 # ======================================================================
 
+def redirect_to_allowed_dept(session, department_id, page):
```

Comment:
From a stylistic perspective, this seems like it could be a decorator.  Currently 100% of the functions which use this have the same first three arguments (``self, session, department_id``), so we should be able to replace e.g.

```python
    @department_id_adapter
    def placeholders(self, session, department_id=None):
        redirect_to_allowed_dept(session, department_id, 'placeholders')
        ...
```

with 

```python
    @department_id_adapter
    @redirect_to_allowed_dept('placeholders')
    def placeholders(self, session, department_id=None):
        ...
```

However, I don't think that's a required/necessary refactor for this PR.

Reply:
Ooh, good point!

Reply:
I think I'm going to skip this just in the interests of time, but yea, we should make this a decorator the next time we look into it